WordPress/src/main/java/org/wordpress/android/ui/posts/PostListMainViewModel.kt

```diff
@@ -65,7 +65,7 @@ import kotlin.coroutines.CoroutineContext
 
 private const val SCROLL_TO_DELAY = 50L
 private const val SEARCH_COLLAPSE_DELAY = 500L
-private val FAB_VISIBLE_POST_LIST_PAGES = listOf(PUBLISHED, DRAFTS)
+private val FAB_VISIBLE_POST_LIST_PAGES = listOf(PUBLISHED, DRAFTS, SCHEDULED, TRASHED)
```

Comment:
I'm a bit on the fence about this change.. mainly because on the scheduled tab, creating a post via the FAB does not actually result in the post being scheduled (i.e. it still requires _manually_ scheduling the post).

Reply:
Keeping this as per discussion, and also make it appear on Pages as well.